test/e2e/go.mod

```diff
@@ -6,12 +6,17 @@ require (
 	github.com/gavv/httpexpect/v2 v2.1.0
 	github.com/gorilla/websocket v1.4.2 // indirect
 	github.com/gruntwork-io/terratest v0.31.2
+	github.com/gxthrj/apisix-ingress-types v0.1.3
+	github.com/gxthrj/apisix-types v0.1.3
+	github.com/gxthrj/seven v0.2.7
```

Comment:
Already imports it into this repo, why references to the old one?

Reply:
This is a point need to discuss. It will ref to a old seven pkg  in go.mod like below:
`github.com/api7/ingress-controller v0.0.0-20210105024109-72e53386de5a`
I think we should merge e2e test into ingress project, unify go.mod.

For this PR, I change to `github.com/api7/ingress-controller ` in go.mod .